index.html

```diff
@@ -11,73 +11,81 @@
-          </a>
-        </li>
-        <li>
-          <a class="social_media_icon">
-            <img src="assets/angellist.svg" alt="angellist icon " />
+          <li>
+            <a class="social_media_icon">
+              <img src="assets/twitter.svg" alt="twitter icon " />
+            </a>
+          </li>
+          <li>
+            <a class="social_media_icon">
+              <img src="assets/Linkedin_icon.svg" alt="Linkedin icon " />
+            </a>
+          </li>
+          <li>
+            <a class="social_media_icon" id="medium_icon">
+              <img src="assets/medium.svg" alt="medium icon " />
+            </a>
+          </li>
+          <li>
+            <a class="social_media_icon">
+              <img src="assets/github.svg" alt="github icon " />
+            </a>
+          </li>
+          <li>
+            <a class="social_media_icon">
+              <img src="assets/angellist.svg" alt="angellist icon " />
+            </a>
+          </li>
+        </ul>
+        <div class="next_page_icon">
+          <a href=".work_section">
+            <svg width="42" height="42" viewBox="0 0 42 42" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <rect x="1" y="1" width="40" height="40" rx="20" fill="white"></rect>
+              <path fill-rule="evenodd" clip-rule="evenodd"
+                d="M14.2929 17.2929C14.6834 16.9024 15.3166 16.9024 15.7071 17.2929L21 22.5858L26.2929 17.2929C26.6834 16.9024 27.3166 16.9024 27.7071 17.2929C28.0976 17.6834 28.0976 18.3166 27.7071 18.7071L21.7071 24.7071C21.3166 25.0976 20.6834 25.0976 20.2929 24.7071L14.2929 18.7071C13.9024 18.3166 13.9024 17.6834 14.2929 17.2929Z"
+                fill="#172B4D"></path>
+              <rect x="1" y="1" width="40" height="40" rx="20" stroke="#DFE1E6"></rect>
+            </svg>
           </a>
-        </li>
-      </ul>
-      <div class="next_page_icon">
-        <a href=".work_section">
-          <svg width="42" height="42" viewBox="0 0 42 42" fill="none" xmlns="http://www.w3.org/2000/svg">
-            <rect x="1" y="1" width="40" height="40" rx="20" fill="white"></rect>
-            <path fill-rule="evenodd" clip-rule="evenodd"
-              d="M14.2929 17.2929C14.6834 16.9024 15.3166 16.9024 15.7071 17.2929L21 22.5858L26.2929 17.2929C26.6834 16.9024 27.3166 16.9024 27.7071 17.2929C28.0976 17.6834 28.0976 18.3166 27.7071 18.7071L21.7071 24.7071C21.3166 25.0976 20.6834 25.0976 20.2929 24.7071L14.2929 18.7071C13.9024 18.3166 13.9024 17.6834 14.2929 17.2929Z"
-              fill="#172B4D"></path>
-            <rect x="1" y="1" width="40" height="40" rx="20" stroke="#DFE1E6"></rect>
-          </svg>
-        </a>
-      </div>
+        </div>
 
-    </section>
+      </section>
+    </div>
+    
     <section class="work_section">
 
-      <h1 class="work_section_title">My Recent <br> Works</h1>
+      <h1 class="work_section_title">My Recent <br class="title_line"> Works</h1>
```

Comment:
- kindly ensure that Your card is ordered like in the Figma design, the respect to the structure of 2 rows with 3 columns (6 projects in total) so that it will make your project stick to the design as required in the project requirement.  you can use this [link ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) to check the design in the Figma :grinning: :grinning: :grinning:

## your design :-1: 
![image](https://user-images.githubusercontent.com/95297251/186784193-1024a6e9-8c9d-4100-ab63-078f9a8719e1.png)
## Figma design :+1: 
![image](https://user-images.githubusercontent.com/95297251/186785782-163759d0-bc8c-4581-a470-09cf0c5398c8.png)